rootex/framework/components/transform_component.cpp

```diff
@@ -78,6 +78,26 @@ void TransformComponent::addTransform(const Matrix& applyTransform)
 	setTransform(getLocalTransform() * applyTransform);
 }
 
+JSON::json TransformComponent::getJSON() const
+{
+	JSON::json j;
```

Comment:
Name json variable properly.

Reply:
'j' stands for JSON, inside every file getJSON() function in a child of Component